ocs_ci/utility/vsphere.py

```diff
@@ -542,6 +542,25 @@ def is_resource_pool_exist(self, pool, dc, cluster):
         """
         return True if self.get_pool(pool, dc, cluster) else False
 
+    def is_resource_pool_prefix_exist(self, pool_prefix, dc, cluster):
+        """
+        Check whether or not resource pool with the provided prefix exist
+
+        Args:
+            pool_prefix (str): The prefix to look for
+            dc (str): Datacenter name
+            cluster (str): Cluster name
+
+        Returns:
+            bool: True if a resource pool with the same name prefix exists, False otherwise
+
+        """
+        cluster_obj = self.get_cluster(cluster, dc)
+        for rp in cluster_obj.resourcePool.resourcePool:
```

Comment:
@ebenahar I guess this will make the troubles for production executions starting with jnk- prefix.
Be aware that we plan to make it probably like j- only in the future to save some characters in cluster name. So worth to use this prefix from some constants and we will change all usage of this jnk- prefix later on from this place and will be easier to change it.  For now I think we can skip this check for such production_jobs_prefix (jnk-).

Reply:
good point. I will add this skip in case of jnk-

Reply:
done - https://github.com/red-hat-storage/ocs-ci/pull/2975/files#diff-9190895310341674a2b7ed1d4b07d5dfR571